contrib/ros/README.md

```diff
@@ -0,0 +1,194 @@
+# rllab.contrib.ros Instructions
+## Introduction
+Most of the current environments for reinforcement learning training(e.g. gym's environments) are not particularly 
+designed for robot learning. Consequently, most of their environments are not for continuous action space task. It is also hard for robot learning researchers to use the same environment interface when they want to test their 
+algorithms on real robots. 
+
+rllab.contrib.ros's environments take a different approach. It's targeted towards providing the same interfaces for training both simulated and real robots. It's developed by Hejia Zhang.
+## Support Platform
+| Robot        | Workstation OS | ROS    | Python | Simulation Platform | 
+|:------------:|:--------------:| :-----:|:------:|:-------------------:|
+| Sawyer       | Ubuntu16.04    | Kinetic| 3      |Gazebo 7.0           |
+### Major python packages dependencies
+- catkin_pkg
+- rospkg
+- defusedxml
+- opencv-python
+## Define bash variables which will be used throughout the tutorial
+### Define the path of python interpreter used in rllab conda env as bash variable
+```bash
+$ source activate rllab3
+$ export RLLAB_PYTHON=`which python`
+$ source deactivate
+```
+### Define your favorite workspace name as bash variable 
```

Comment:
merge these three sub headings (defining variables) into one set of commands in a shell block.